internal/commands/sastrealtime/sastconfiguration/sast-linux.go

```diff
@@ -0,0 +1,16 @@
+//go:build linux
+
+package sastconfiguration
+
+import (
+	"github.com/checkmarx/ast-cli/internal/services/osinstaller"
+)
+
+var Params = osinstaller.InstallationConfiguration{
```

Comment:
The values in the configuration are duplicate in the sast-mac, sast-linux and sast-windows...
it's better to have them as constants and if and when they change - you'd only have to change them once.
Notice we have a constants folder, you can add a new file there.